axlearn/common/measurement.py

```diff
@@ -41,6 +42,26 @@ class Event(enum.Enum):
     END_CUSTOM_BADPUT_EVENT = "END_CUSTOM_BADPUT_EVENT"
 
 
+class EventType(enum.Enum):
+    """Event to be recorded.
+
+    Attributes:
+        JOB: Start and end of the job.
+        STEP: Start of a training step. Should be recorded with `step` as a positional arg.
+        ACCELERATOR_INIT: Start and end of accelerator mesh initialization.
+        TRAINING_PREPARATION: Start and end of training preparation.
+        DATA_LOADING: Start and end of data loading.
+        CUSTOM_BADPUT_EVENT: Start and end of custom badput events.
+    """
+
+    JOB = "job"
+    STEP = "step"
+    ACCELERATOR_INIT = "tpu_init"
+    TRAINING_PREPARATION = "training_preparation"
+    DATA_LOADING = "data_loading"
+    CUSTOM_BADPUT_EVENT = "custom_badput_event"
```

Comment:
Are we actually aiming to keep this `EventType` class (as well as `Event`), or is this leftover code?

I'm not seeing this used anywhere in the trainer or tested in `axlearn/common/measurement_test`.
If it's only used in goodput recorder, maybe move the abstraction into goodput recorder implementation?